Misc/NEWS.d/next/Library/2025-03-08-17-07-00.gh-issue-88473.qg23g8.rst

```diff
@@ -0,0 +1 @@
+Implement a fast path for date types in :func:`datetime.date.today`
```

Comment:
I would say for "date objects" + link on "datw" and add a second sentence that proper subclasses of date retain their previous performance. Maybe add also that the improvement factor is 5.